app/routes/revocation.py

```diff
@@ -313,3 +313,50 @@ async def get_pending_revocations(
 
     bound_logger.debug("Successfully fetched pending revocations.")
     return PendingRevocations(pending_cred_rev_ids=result)
+
+
+@router.put(
+    "/fix-revocation-registry/{revocation_registry_id}",
+    summary="Fix Revocation Registry Entry State",
+)
+async def fix_revocation_registry_entry_state(
+    revocation_registry_id: str,
+    apply_ledger_update: bool = False,
+    auth: AcaPyAuth = Depends(acapy_auth_from_header),
+) -> RevRegWalletUpdatedResult:
+    """
+    Fix Revocation Registry Entry State
+    ---
+    Fix the revocation registry entry state for a given revocation registry ID.
+
+    If issuer's revocation registry wallet state is out of sync with the ledger,
+    this endpoint can be used to fix/update the ledger state.
+
+    Path Parameters:
+    ---
+        revocation_registry_id: str
+            The ID of the revocation registry for which to fix the state
+
+    Query Parameters:
+    ---
+        apply_ledger_update: bool
+            Set to True to apply the ledger update
```

Comment:
What does False do? Would be good to just add a note, like False applies to wallet state, but not ledger (idk the specifics)